### ISSUE TITLE
Online playground on error click add scroolToLine

### ISSUE THREAD

**Issue by mingodad** [yhirose/cpp-peglib#198]
On the online playground when we click an error message if we add `scroolToLine` then the editor does exactly that:

index.js:
```
...
// Event handing in the info area
function makeOnClickInInfo(editor) {
  return function () {
    const el = $(this);
    editor.navigateTo(el.data('ln') - 1, el.data('col') - 1);
    editor.scrollToLine(el.data('ln') - 1, true, false, null);  /////!!!!! <<<<<  add it here
    editor.focus();
  }
};
...
```

**Comment by mingodad** [comment]
Also adding a hover background color on error messages make then more evident:
style.css:
```
...
.editor-info li:hover{
  background-color: yellow;
}
```

**Comment by yhirose** [comment]
Fixed at #199.
